nsxt/resource_nsxt_vpc_connectivity_profile_test.go

```diff
@@ -0,0 +1,210 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestVpcConnectivityProfileCreateAttributes = map[string]string{
+	"display_name":        getAccTestResourceName(),
+	"description":         "terraform created",
+	"enable":              "false",
+	"enable_default_snat": "false",
+}
+
+var accTestVpcConnectivityProfileUpdateAttributes = map[string]string{
+	"display_name":        getAccTestResourceName(),
+	"description":         "terraform updated",
+	"enable":              "false",
```

Comment:
Nit: if parameters are same for create and update, maybe it's worthwhile to drop them

Reply:
I hope those will change once NSX API is fully functional